### ISSUE TITLE
dnd-draggable == "true" prevents ng-click 

### ISSUE THREAD

**Issue by Dimanoid** [Tuch/angular-dnd#34]
Look at the example, if dnd-draggable set to true ng-click event never fired.
http://embed.plnkr.co/w5vP9BK3xVztYhVBe2OI/preview

You need to implement something like dnd-drag-delay option or dnd-drag-delta to start dragging after a few ms after onmousedown or after cursor were moved a few pixels away from start point with button hold pressed. Something similar is done in angular-ui-tree https://github.com/angular-ui-tree/angular-ui-tree

**Comment by Tuch** [comment]
@Dimanoid  hmm, in my case this working. Maybe i do something wrong?

**Comment by Dimanoid** [comment]
Stange but I cant reproduce it too on other computer.
Sorry for disturbance. 

Btw I think that has an option to delay start dragging is good anyway... Count it as a feature request :)

**Comment by Tuch** [comment]
@Dimanoid hmm, i think yes, but need more info about this point :) can you describe it?

**Comment by Dimanoid** [comment]
Please check JQueryUI API, it has both options.
http://api.jqueryui.com/draggable/#option-delay
http://api.jqueryui.com/draggable/#option-distance
They are great to prevent accident start drag.

**Comment by hebedombiu** [comment]
I have the same problem. It begins drag when I click on element. And I can't check checkbox on child div.
Win8, Chrome
Which side I have to look, to fix that?

**Comment by Dimanoid** [comment]
Yep its really annoying when you start dragging instead of just click because of cursor micro-movement. Can you please fix it anytime soon? Those new options will resolve this nicely.

**Comment by Tuch** [comment]
@hebedombiu, @Dimanoid You mean like 'tollerance movement' option?

**Comment by k-one-o-two** [comment]
I think that I've found the problem on Chrome in Win10, this code seems to fix it:
```javascript
mousemove: function (event) {
                if (event.movementX != 0 && event.movementY != 0) {
                    this.manipulator.progress(event);
                }
            }
```
The problem is that Chrome fires `mousemove` event with zero movents, which leads to the error described above.

**Comment by Tuch** [comment]
@k-one-o-two good idea, but bad supporting for movementX property...
